pycytominer/cyto_utils/cells.py

```diff
@@ -397,6 +397,34 @@ def get_subsample(self, df=None, compartment="cells", rename_col=True):
 
         self.is_subset_computed = True
 
+    def is_feature_col(self, col):
+        """Check if column is a feature."""
+        return (
+            col.startswith("Cell")
+            or col.startswith("Cytoplasm")
+            or col.startswith("Nuclei")
+        )
+
+    def count(self, table):
+        """Count total number of rows for a table."""
+        (num_rows,) = next(self.conn.execute(f"SELECT COUNT(*) FROM {table}"))
+        return num_rows
+
+    def get_columns(self, table):
```

Comment:
more descriptive name please.

For this, how about `get_sql_table_col_names()`?